documentation/testing/performance-test-plan/Establisment_API_Peak-load-testing-plan.md

```diff
@@ -0,0 +1,77 @@
+﻿# API Peak Load Testing Plan
+
+## Description
+This test simulates a gradual increase in the number of users to evaluate the system's performance at its expected peak load. The goal is to ensure that the API endpoints can handle the load without significant degradation in performance metrics.
+
+## Procedure
+
+### Determine Capacity
+Calculate the total number of requests that each API endpoint can handle simultaneously.
+
+### Ramp-Up Phase
+For each endpoint, gradually increase the number of concurrent users over a ramp-up period of 5-10 seconds.
```

Comment:
I wouldn't say that ramping to 50000 users in 10 secs is gradual. :)

Reply:
updated the plan and ready for second review.